contrib/jenkins_tests/test.sh

```diff
@@ -80,6 +81,14 @@ for test_link in $test_ip_list; do
 	done
 done
 
+for test_link in $test_uds_list; do
+	test_in="uds"
+	test_name="uds_test"
+	$timeout_exe ${WORKSPACE}/tests/verifier/verifier.pl -a ${test_app} \
+            -t ${test_in} -s ${test_link} -l ${test_dir}/${test_name}.log \
+            --progress=0
+done
+
```

Comment:
Only thing you need is just 
```
test_ip_list="local:/tmp/sockperf-test-$$"
test_list="uds"

for test_link in $test_ip_list; do
	for test in $test_list; do
<the same as above>
        done
done
```

Reply:
`test_ip_list` and `test_list` are in use for the original flow, 
so I can create `test_uds_list` and `test_uds_path` to distinguish between UDS and IP, and as we spoke to create new for loop for UDS separately

Reply:
e.g 
`test_uds_list="uds"
test_uds_path="local:/tmp/sockperf-test-$$"


for test_link in $test_uds_list; do
	for test in $test_uds_path; do
<the same as above>
        done
done`

Reply:
> `test_ip_list` and `test_list` are in use for the original flow, so I can create `test_uds_list` and `test_uds_path` to distinguish between UDS and IP, and as we spoke to create new for loop for UDS separately

Just reassign these variables